fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/RecordCursorTest.java

```diff
@@ -328,20 +328,17 @@ public void pipelineContinuationTest() throws Exception {
         List<Integer> expected = ints.stream().flatMap(o -> ints.stream().map(i -> o * 100 + i)).collect(Collectors.toList());
 
         Function<byte[], RecordCursor<Integer>> outerFunc = cont -> RecordCursor.fromList(ints, cont);
-        Function<Integer, RecordCursor<Integer>> innerFunc1 = outer -> RecordCursor.fromList(ints)
+        BiFunction<Integer, byte[], RecordCursor<Integer>> innerFunc = (outer, cont) -> RecordCursor.fromList(ints, cont)
                 .map(inner -> outer.intValue() * 100 + inner.intValue());
-        assertEquals(expected, outerFunc.apply(null).flatMapPipelined(innerFunc1, 1).asList().join());
-
-        BiFunction<Integer, byte[], RecordCursor<Integer>> innerFunc2 = (outer, cont) -> RecordCursor.fromList(ints, cont)
-                .map(inner -> outer.intValue() * 100 + inner.intValue());
-        assertEquals(expected, RecordCursor.flatMapPipelined(outerFunc, innerFunc2, null, 1).asList().join());
+        assertEquals(expected, RecordCursor.flatMapPipelined(outerFunc, (outer, cont) -> RecordCursor.fromList(ints)
```

Comment:
Was it intentional to leave out `cont` in the the call to `RecordCursor.fromList(ints)`? And if it wasn't, should it be `innerFunc` here?

Reply:
Oh, not especially. I was just adapting the original code to the new function and missed a simplification. I changed it to `innerFunc`, which is cleaner.